conftest.py

```diff
@@ -76,46 +80,81 @@ def sample_dag():
         session.query(TI).delete()
 
 
+def populate_table(path: str, table: Table, hook: BaseHook) -> None:
+    """
+    Populate a csv file into a sql table
+    :param path: path to csv file
+    :param table: Astro table object
+    :return: None
```

Comment:
We can exclude this since we have it in TypeHints

Reply:
Done